From 5d489188a4e6e4aa0c63e0e92d89a9474a802e3d Mon Sep 17 00:00:00 2001
From: Mark Brown <broonie@linaro.org>
Date: Sun, 18 May 2014 18:36:28 +0100
Subject: [PATCH] configs: Make cpufreq not set for RT

Not sure this is going to do the right thing but it fixes errors that
the CI sees.  Why isn't the RT testing just setting the performance
governor anyway?

Signed-off-by: Mark Brown <broonie@linaro.org>
---
 linaro/configs/preempt-rt.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linaro/configs/preempt-rt.conf b/linaro/configs/preempt-rt.conf
index f8873a6..98e036d 100644
--- a/linaro/configs/preempt-rt.conf
+++ b/linaro/configs/preempt-rt.conf
@@ -1,4 +1,4 @@
 CONFIG_PREEMPT=y
 CONFIG_PREEMPT_RT_FULL=y
 CONFIG_SLUB=y
-CONFIG_CPU_FREQ=n
+# CONFIG_CPU_FREQ is not set
-- 
1.8.0

